app/assets/javascripts/actions/training_modification_actions.js

```diff
@@ -0,0 +1,35 @@
+import { API_FAIL } from '../constants/api';
+import { addNotification } from '../actions/notification_actions';
+import logErrorMessage from '../utils/log_error_message';
+import request from '../utils/request';
+
+const createLibraryPromise = async (library, setSubmitting, modalHandler) => {
+    const response = await request('/training_library', {
+      method: 'POST',
+      headers: {
+        'Content-Type': 'application/json',
+      },
+      body: JSON.stringify(library),
+    });
+    setSubmitting(false);
+    modalHandler(false);
```

Comment:
`modalHandler` [doesn't take any arguments](https://github.com/WikiEducationFoundation/WikiEduDashboard/pull/5810/files#diff-ebc286207cbdcfed824d6b79fd1d3086737bcd6cc94880d79a31181968f20943R11-R14) — I think this is in error?

Reply:
yes, now i made it correct